src/engine/strat_engine/crypt/shared.rs

```diff
@@ -100,9 +101,10 @@ fn key_desc_to_passphrase(key_description: &KeyDescription) -> StratisResult<Siz
 // in memory (this is automatically done when formatting a LUKS2 device).
 pub fn add_keyring_keyslot(
     device: &mut CryptDevice,
+    token_slot: Option<u32>,
```

Comment:
Need to update Postcondition to match argument.

Also, I believe that the first error message that is returned, "Failed to initialize keyslot with existing Clevis key" becomes slightly stale with this PR. It should be something more like "Failed to initialize keyslot with existing key", as it is no longer guaranteed to be a Clevis key that you tried to use.

Reply:
This should be resolved.